test/oatpp/web/FullTest.cpp

```diff
@@ -292,6 +292,11 @@ void FullTest::onRun() {
         OATPP_ASSERT(dto->testValue == "my_test_header");
       }
 
+      // { // test silent close
```

Comment:
Client should just throw an error in this situation (I mean it should be already working:)):

```cpp
try{
      auto response = client->silentClose();
      OATPP_ASSERT(response == nullptr);
} catch (const std::runtime_erro& e) {
   ...
}
```